### DISCUSSION TITLE
Working example of st_draw_features

### DISCUSSION THREAD

**Discussion by julianblue** [opengeos/leafmap#499] [Q&A] [answered]
Hi @giswqs ,

is there an example on how to use https://leafmap.org/foliumap/?h=st_draw_features#leafmap.foliumap.Map.st_draw_features ?

I am not sure which streamlit component is meant. 

thanks in advance

**Comment by giswqs** [comment, marked as answer]
It uses [streamlit-folium](https://github.com/randyzwitch/streamlit-folium). 

I no longer use streamlit. Solara is so much better for building geospatial appliations with be-directional communication. See this example. 
https://github.com/opengeos/solara-geospatial

**Reply by julianblue** [reply]
Thanks @giswqs . Will look into it.
